README.md

```diff
@@ -73,80 +75,86 @@ We also have a detailed [tutorial](tutorial.md) that has been given in a couple
-
-4.  This will create a specific directory structure.
-
-    For example, for a project with the following variables:
-
-    ```
-    project_name [Python Template]: PROJECT_NAME
-    project_slug [python-template]: PROJECT_SLUG
-    package_name [python_template]: PACKAGE_NAME
-    ```
-
-    we will get a project folder named `PROJECT_SLUG`, structured like this:
-
-    ```
-    PROJECT_SLUG
-    ├── ...
-    ├── README.md
-    ├── pyproject.toml
-    ├── src
-    │   └── PACKAGE_NAME
-    │       └── __init__.py
-    └── tests
-        └── test_dummy.py
-    ```
-
-    And the `PROJECT_NAME` will appear in the README.md as the human-readable
-    name of the project.
-
-    ```
-    cat PROJECT_SLUG/README.md
-    # PROJECT_NAME
-    ...
-    ```
-
-5.  To work on your project, initialise a Git repository and _install_ it in
-    editable mode.
-    ```
-    cd PROJECT_SLUG
-    git init
-    python -m pip install -e ".[dev]"
-    ```
-6.  Build your package
-    ```
-    python -m build
-    ```
+1. Install [cookiecutter] in a Conda or `venv` environment (commented lines for
+   Conda example).
+
+   ```sh
+   # conda create --channel conda-forge --name new-env-name
+   # conda activate new-env-name
+   # conda install pip
+   pip install cookiecutter
+   ```
+
+2. Run cookiecutter in the desired directory
+
+   ```sh
+   cookiecutter gh:ucl-arc/python-tooling
+   ```
+
+   If you have this repo locally (this may be the case if you are developing),
+   you can run the following:
+
+   ```sh
+   cookiecutter /path/to/your/checkout/of/python-tooling
+   ```
+
+3. A series of questions will pop up to configure the project. Type the answer
+   or hit return to use the default option (shown in parenthesis).
+
+   Note that these project variables are defined in the `cookiecutter.json`
+   file.
+
+4. This will create a specific directory structure.
+
+   For example, for a project with the following variables:
+
+   ```yaml
+   project_name [Python Template]: PROJECT_NAME
+   project_slug [python-template]: PROJECT_SLUG
+   package_name [python_template]: PACKAGE_NAME
+   ```
+
+   we will get a project folder named `PROJECT_SLUG`, structured like this:
+
+   ```sh
+   PROJECT_SLUG
+   ├── ...
+   ├── README.md
+   ├── pyproject.toml
+   ├── src
+   │   └── PACKAGE_NAME
+   │       └── __init__.py
+   └── tests
+       └── test_dummy.py
+   ```
+
+   And the `PROJECT_NAME` will appear in the README.md as the human-readable
+   name of the project.
+
+   ```sh
+   cat PROJECT_SLUG/README.md
+   # PROJECT_NAME
+   ...
+   ```
+
+5. To work on your project, initialise a Git repository and _install_ it in
+   editable mode.
+
+   ```sh
+   cd PROJECT_SLUG
+   git init
+   python -m pip install -e ".[dev]"
+   ```
+
+6. Build your package
+
+   ```sh
+   python -m build
+   ```
 
 ## Notes for developers
 
-<details>
-<summary>Click to expand...</summary>
+<details><summary>Click to expand...</summary> <!-- markdownlint-disable-line MD033 -->
```

Comment:
Combining these, is just so we only have to have one `markdownlint-disable-line`

Reply:
That's the second time I've thought that these instructions should be moved/merged into CONTRIBUTING.md.

- #466